ext/date/php_date.c

```diff
@@ -1948,10 +1954,20 @@ static void php_timezone_to_string(php_timezone_obj *tzobj, zval *zv)
 	}
 }
 
+void date_timezone_object_to_hash(php_timezone_obj *tzobj, HashTable *props)
+{
+	zval zv;
+
+	ZVAL_LONG(&zv, tzobj->type);
+	zend_hash_str_update(props, "timezone_type", sizeof("timezone_type")-1, &zv);
+
+	php_timezone_to_string(tzobj, &zv);
+	zend_hash_str_update(props, "timezone", sizeof("timezone")-1, &zv);
```

Comment:
Nit: maybe while touching this already use ``strlen()`` instead of ``sizeof()-1``?